src/microbe_stage/PatchMapNameGenerator.cs

```diff
@@ -66,8 +51,7 @@ public string Next(Random random)
         // Convert first letter to uppercase
         name[0] = char.ToUpper(name[0], CultureInfo.InvariantCulture);
 
-        // Continent name is the name without the genitive
```

Comment:
Could you put this comment back, I think it's important to keep for future understandability?